tests/test.py

```diff
@@ -2021,6 +2103,100 @@ def test_build_container_invalid_container_limits(self):
             })
         )
 
+    ###################
+    #  VOLUMES TESTS  #
+    ###################
+
+    @pytest.mark.skipif(docker.utils.compare_version(
+        '1.21', docker.constants.DEFAULT_DOCKER_API_VERSION
+    ) < 0, reason="API version too low")
```

Comment:
This very check is rather verbose (and needs to be repeated in a few places).  You can actually do something like this to re-use the mark (https://pytest.org/latest/skipping.html#skipif):

```python

def requires_api_version(version):
    return pytest.mark.skipif(
        docker.utils.compare_version(
            version,
            docker.constants.DEFAULT_DOCKER_API_VERSION
        ) < 0,
        reason="API version too low"
    )


@requires_api_version('1.21')
def test...
```